tests/sequence_labelling/preprocess_test.py

```diff
@@ -132,3 +132,31 @@ def test_should_select_features(self):
         features_length = len(preprocessor.features_indices)
         assert features_length == 1
         assert all_close(features_transformed, [[[1], [2], [3]]])
+
+    @pytest.fixture(scope="function")
```

Comment:
you shouldn't need that here, at least for `tmp_path`.

Reply:
This was not working anyway... I remove that stuff